src/workloads/execution/CumulativeWindows.yml

```diff
@@ -0,0 +1,155 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    # Small collection size to keep from spilling to disk during sorts.
+    DocumentCount: 10000
+    BatchSize: &batchSize 10000
+    Document:
+      t: {^RandomDate: {min: "2020-01-01", max: "2021-01-01"}}
+      x: {^RandomDouble: {distribution: normal, mean: 0, sigma: 3}}
+      y: {^RandomDouble: {distribution: normal, mean: 1, sigma: 3}}
+      z: {^RandomString: {length: 8}}
```

Comment:
This one looks unused, was it intentional? It would be fine to leave in with a comment, given its likely that real world use cases will have more fields than just the ones used by WFs.

Reply:
Yes that was the intention. I can leave a comment to that effect.